app/controllers/submit.js

```diff
@@ -1133,68 +1208,35 @@ export default Controller.extend(Analytics, BasicsValidations, NodeActionsMixin,
         }
     },
 
-    _setNodePreprints(preprints) {
-        this.set('existingPreprints', preprints);
-        if (preprints.toArray().length > 0) { // If node already has a preprint
-            const preprint = preprints.toArray()[0]; // TODO change after branded finished
-            if (!(preprint.get('isPublished'))) { // Preprint exists in abandoned state.
-                this.set('abandonedPreprint', preprint);
-            }
-        }
-    },
-
-    _sendToPreprintStartOrAbandon() {
-        this.send(this.get('abandonedPreprint') ? 'resumeAbandonedPreprint' : 'startPreprint');
```

Comment:
We don't care if there are abandoned preprints anymore, this was before the one preprint per node constraint.